From 959a9c3b259e50286a029634cb1bfde66daed34e Mon Sep 17 00:00:00 2001
From: Jason Lowe <jlowe@apache.org>
Date: Tue, 23 Aug 2016 14:51:29 +0000
Subject: [PATCH 2083/2848] HDFS-9745. TestSecureNNWithQJM#testSecureMode
 sometimes fails with timeouts. Contributed by
 Xiao Chen (cherry picked from commit
 126d165efd80e266a8309241f3cf059e358f5019)

(cherry picked from commit cec6083055cc0a35f8322370cdd8e3e1392f43ad)

 Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt

Change-Id: I67e9716d2ef9c7883d94c5922bf7ecc0fa8bff26
---
 .../hadoop/hdfs/qjournal/TestSecureNNWithQJM.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/TestSecureNNWithQJM.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/TestSecureNNWithQJM.java
index f503319..0a2270d 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/TestSecureNNWithQJM.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/TestSecureNNWithQJM.java
@@ -76,7 +76,7 @@
   private MiniJournalCluster mjc;
 
   @Rule
-  public Timeout timeout = new Timeout(30000);
+  public Timeout timeout = new Timeout(180000);
 
   @BeforeClass
   public static void init() throws Exception {
-- 
1.7.9.5

